### PR TITLE
Add semantic search throughput blog

### LINKED ISSUE #1270: [Blog] A benchmarking study on semantic search with OpenSearch
**Describe the blog post your would like to write**
A benchmarking study based on the recently launched (experimental) neural-search plugin, with latency/throughput comparison between different traffic/host configuration and etc.

**What is the title of the blog post?**
A benchmarking study on semantic search with OpenSearch

**Who are the authors?**
Sean Zheng, Zane Niu, Navneet Verma, Junshen Wu, Charlie Yang, Yaliang Wu, Vamshi Vijay

**What is the proposed posting date?**
02/06/2023 --> 03/27/2023

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Replaces #1766 

Fixes #1270 

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by natebower — COMMENTED** [review]
@pajuric @kolchfa-aws Because this was pushed to my queue, I just finished editing it prior to it being closed. Given the time I spent on it, can we please apply my edits, regardless of where it's published? Thanks!

**Comment by kolchfa-aws** [comment]
Reopening to apply editorial comments

File: _posts/2023-11-22-semantic-search-performance.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+
+Say that a baker and a cashier take the same amount of time to process a request and this amount is 5 minutes per customer. The throughput at the bakery counter with 5 bakers, as before, is 1 customer/min. The throughput at the checkout with 5 cashiers is also 1 customer/min. But what is the throughput of the entire bakery? The throughput of the entire system is also 1 customer/min! To understand this, imagine you are observing the exit door of the bakery. The rate at which customers come out of the exit door is the rate at which the last step of the sequence (cashiers at the checkout) processes requests. 
+
+What if a cashier takes far less time than the baker? Say the cashier takes 2.5 minutes per customer while the baker takes 5 minutes per customer. In this case, the throughput at the bakery counter is, as before, 1 customer/min. But what is the throughput at the checkout? It might seem that since each cashier serves 60 minutes / 2.5 minutes = 24 customers per hour, and there are 5 cashiers in total, the throughput should be 24 * 5 = 120 customers per hour (or 2 customers per minute). But this calculation is wrong. Do you see why?
```
[the remark this answers is not in the document]
Comment (kolchfa-aws) [reply _posts/2023-11-22-semantic-search-performance.md]: Corrected the units.

File: _posts/2023-11-22-semantic-search-performance.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+
+|Data node	|ML node	|p50 -- Profile API latency (seconds)	|p50 -- CPU utilization (data node, ML node)	|Actual throughput (p50)	|Estimated throughput	|
+|---	|---	|---	|---	|---	|---	|
+|1 r6g.4xlarge: 16 vCPU	|10 c6g.12xlarge: 48 vCPU	|3.878	|(7, 99)	|261 docs/sec	|247 docs/sec	|
+|1 r6g.4xlarge: 16 vCPU	|1 c6g.12xlarge: 48 vCPU	|3.878	|(16, 99)	|27 docs/sec	|24.7 docs/sec	|
+|1 m5.2xlarge: 8vCPU	|5 m5.2xlarge: 8vCPU	|1.965	|(8, 99)	|39.62 docs/sec	|40.7 docs/sec	|
+|1 m5.2xlarge: 8vCPU	|20 m5.2xlarge: 8vCPU	|1.914	|(56, 95)	|154.24 docs/sec	|167.2 docs/sec	|
+|1 m5.2xlarge: 8vCPU	|20 m5.2xlarge: 8vCPU	|2.007	|(6, 97)	|159.2 docs/sec	|159.4 docs/sec	|
+
+_Comparison of the actual throughput with the estimated throughput for different cluster settings (ingestion throughput $$I$$ is calculated using formula $$(\ref{1})$$ and latency $$t_{embed}$$ is measured using the Profile API)​._
```
[the remark this answers is not in the document]
Comment (kolchfa-aws) [reply _posts/2023-11-22-semantic-search-performance.md:181]: This is the caption for the table. In parentheses, there are a couple of clarifications how the throughput and latency are calculated/measured. LMK if you think it should be reworded.

[the remark this answers is not in the document]
Comment (natebower) [reply _posts/2023-11-22-semantic-search-performance.md:181]: Got it. In that case there's just a missing comma.

[the remark this answers is not in the document]
Comment (natebower) [reply _posts/2023-11-22-semantic-search-performance.md:181]:
```suggestion
_Comparison of the actual throughput with the estimated throughput for different cluster settings (ingestion throughput $$I$$ is calculated using formula $$(\ref{1})$$, and latency $$t_{embed}$$ is measured using the Profile API)​._
```

**Comment by natebower** [comment]
Thanks @kolchfa-aws! @pajuric Can you ensure that this edited version is passed to the AWS side of the house? Thanks!

**Comment by kolchfa-aws** [comment]
Closing because this blog will be published on AWS

**Pull request closed without merging**